mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -492,7 +492,7 @@ def delete_tsdb_records(
         # Split the endpoint ids into chunks to avoid exceeding the v3io-engine filter-expression limit
         for i in range(0, len(endpoint_ids), V3IO_FRAMESD_MEPS_LIMIT):
             endpoint_id_chunk = endpoint_ids[i : i + V3IO_FRAMESD_MEPS_LIMIT]
-            filter_query = f"endpoint_id IN({str(endpoint_id_chunk)[1:-1]}) "
+            filter_query = f"endpoint_id IN({', '.join(endpoint_id_chunk)}) "
```

Comment:
Just checking that you verified it works. It’s not exactly the same. Before your change, the result included a backtick (`):

`endpoint_id IN('abc', 'def', 'ghi') `

but after your change:
`endpoint_id IN(abc, def, ghi) `